onchain/rollups/contracts/inputs/IInputBox.sol

```diff
@@ -32,6 +32,7 @@ interface IInputBox {
     /// @param _input The contents of the input
     /// @return The hash of the input plus some extra metadata
     /// @dev MUST fire an `InputAdded` event accordingly.
+    ///      Input larger than machine limit will raise `InputSizeExceedsLimit` error.
```

Comment:
In this case, the problem of an input not fitting in an off-chain machine is orthogonal to the implementation of the input box contract. That is to say that every implementation of the input box should raise this error if an input is too big, and, therefore, that the definition of this error (currently in `LibInput`) should be moved to this interface.